ha/setup/readme.txt

```diff
@@ -0,0 +1,2 @@
+This directory contains the scripts for R2 for e.g. creating and configuring the cluster.
```

Comment:
I would suggest to use markdown here, so it can be displayed in web-interface when folder is opened.

Reply:
done